diff --git a/pyop2/configuration.py b/pyop2/configuration.py
index 2971771..5f8c0a6 100644
--- a/pyop2/configuration.py
+++ b/pyop2/configuration.py
@@ -79,9 +79,9 @@ class Configuration(dict):
     cache_dir = os.path.join(gettempdir(), "pyop2-cache-uid%s" % os.getuid())
     DEFAULTS = {
         "cc":
-            ("PYOP2_CC", str, ""),
+            ("PYOP2_CC", str, "@mpi@/bin/mpicc"),
         "cxx":
-            ("PYOP2_CXX", str, ""),
+            ("PYOP2_CXX", str, "@mpi@/bin/mpicxx"),
         "ld":
             ("PYOP2_LD", str, ""),
         "cflags":
