diff --git a/firedrake/__init__.py b/firedrake/__init__.py
index a888068..6de1f44 100644
--- a/firedrake/__init__.py
+++ b/firedrake/__init__.py
@@ -1,19 +1,8 @@
 import firedrake_configuration
 import os
 import sys
-config = firedrake_configuration.get_config()
-if "PETSC_DIR" in os.environ and not config["options"]["honour_petsc_dir"]:
-    if os.environ["PETSC_DIR"] != os.path.join(sys.prefix, "src", "petsc")\
-       or os.environ["PETSC_ARCH"] != "default":
-        raise ImportError("PETSC_DIR is set, but you did not install with --honour-petsc-dir.\n"
-                          "Please unset PETSC_DIR (and PETSC_ARCH) before using Firedrake.")
-elif "PETSC_DIR" not in os.environ and config["options"]["honour_petsc_dir"]:
-    raise ImportError("Firedrake was installed with --honour-petsc-dir, but PETSC_DIR is not set.\n"
-                      "Please set PETSC_DIR (and PETSC_ARCH) before using Firedrake.")
-elif not config["options"]["honour_petsc_dir"]:  # Using our own PETSC.
-    os.environ["PETSC_DIR"] = os.path.join(sys.prefix, "src", "petsc")
-    os.environ["PETSC_ARCH"] = "default"
-del config
+
+os.environ["PETSC_DIR"] = os.environ.get("PETSC_DIR", "@petsc_dir@")
 
 # Set up the cache directories before importing PyOP2.
 firedrake_configuration.setup_cache_dirs()
diff --git a/setup.py b/setup.py
index fae92d9..fa31b80 100644
--- a/setup.py
+++ b/setup.py
@@ -24,7 +24,7 @@ except ImportError:
 
 def get_petsc_dir():
     try:
-        petsc_dir = os.environ["PETSC_DIR"]
+        petsc_dir = os.environ.get("PETSC_DIR", "@petsc_dir@")
         petsc_arch = os.environ.get("PETSC_ARCH", "")
     except KeyError:
         try:
